### PR TITLE
[AWS] Document EKS snap and its updates

### PR DESCRIPTION
**Pull request by woky**

### CODE REVIEW

File: aws/aws-explanation/eks-kubelet-snap.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+See the `official snap documentation <https://snapcraft.io/docs/channels>`_ for more information
+about the concept of channels.
+
+Automatic snap updates
+~~~~~~~~~~~~~~~~~~~~~~
+
+Note that in EKS worker node images, automatic background updates of snaps are disabled. To enable
+automatic background updates of snaps run the following command in the worker node::
+
+    sudo snap refresh --unhold
```
Comment (toabctl) [inline aws/aws-explanation/eks-kubelet-snap.rst:22]: nit: maybe add a note that this unholds everything? including `kubectl` and `aws-cli` ?

File: aws/aws-explanation/eks-kubelet-snap.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Updates of kubelet-eks snap shouldn't affect running workloads.
```
Comment (rpocase) [inline aws/aws-explanation/eks-kubelet-snap.rst:24]:
```suggestion
Updates of kubelet-eks snap shouldn't affect running workloads, but kubeapi access for the node is expected to be unavailable for up to 30 seconds.
```

**Review by rpocase — CHANGES_REQUESTED** [review]
open to wording on requests

File: aws/aws-explanation/eks-kubelet-snap.rst (hunk 2 above)
Comment (woky) [reply aws/aws-explanation/eks-kubelet-snap.rst]: I've replaced kubeapi with _ Kubernetes API_.

**Review by k-dimple — APPROVED** [review]
This looks good. Is it okay if I put it under the 'reference' section instead of 'explanation'? It seems to contain information that is more appropriate for 'work' rather than 'study'.

**Comment by woky** [comment]
@k-dimple Sounds good to me.

**Pull request merged**